The PreCommit Python ML tests with ML deps installed job is flaky

The PreCommit Python ML tests with ML deps installed is failing over 50% of the time.
Please visit https://github.com/apache/beam/actions/workflows/beam_PreCommit_Python_ML.yml?query=is%3Afailure+branch%3Amaster to see all failed workflow runs.
See also Grafana statistics: http://metrics.beam.apache.org/d/CTYdoxP4z/ga-post-commits-status?orgId=1&viewPanel=7&var-Workflow=PreCommit%20Python%20ML%20tests%20with%20ML%20deps%20installed

```
chunks = [Chunk(content=Content(text='This is a simple test document. It has multiple'), id='0791e41b-2607-49af-938c-36a2ee8804...c splitting.'), id='63f67fb4-8347-4c3a-8829-673602f5a439', index=2, metadata={'source': 'simple.txt'}, embedding=None)]

    def check_token_lengths(chunks):
      for chunk in chunks:
        # Verify each chunk's token length is within limits
        num_tokens = len(tokenizer.encode(chunk.content.text))
        if not num_tokens <= 10:
>         raise BeamAssertException(
              f"Chunk has {num_tokens} tokens, expected <= 10")
E         apache_beam.testing.util.BeamAssertException: Chunk has 12 tokens, expected <= 10 [while running 'assert_that/Match']

apache_beam/ml/rag/chunking/langchain_test.py:172: BeamAssertException```

--- a/sdks/python/tox.ini
+++ b/sdks/python/tox.ini
@@ -108,6 +108,7 @@ commands =
 deps =
   pip==25.0.1
   accelerate>=1.6.0
+  onnx<1.19.0
 setenv =
 extras = test,gcp,dataframe,ml_test
 commands =

